linty_fresh/linters/android.py

```diff
@@ -4,18 +4,22 @@
 from linty_fresh.problem import Problem
 
 
-def parse(contents: str) -> Set[Problem]:
+def parse(contents: str, **kwargs) -> Set[Problem]:
     result = set()  # type: Set[Problem]
     try:
         root = ElementTree.fromstring(contents)
     except ElementTree.ParseError:
         return result
     for issue in root.findall('issue'):
         location = issue.getchildren()[0]
-        result.add(Problem(
-            location.get('file'),
-            location.get('line', '0'),
-            '{}: {}'.format(
-                issue.get('summary'),
-                issue.get('message'))))
+        if issue.get('severity') == 'Warning' and not kwargs.get(
```

Comment:
You can actually declare `fail_warnings` as a keyword argument now instead of pulling it out of the kwargs map.